tracing-error/src/backtrace.rs

```diff
@@ -120,6 +120,47 @@ impl SpanTrace {
             }
         });
     }
+
+    /// Returns the status of this SpanTrace, indicating whether this backtrace request was unsupported, empty, or a span trace was actually captured.
```

Comment:
I took a crack at making this a little clearer, what do you think?
```suggestion
    /// Returns the status of this `SpanTrace`.
    ///
    /// The status indicates one of the following:
    /// * the current subscriber does not support capturing `SpanTrace`s
    /// * there was no current span, so a trace was not captured
    /// * a span trace was successfully captured
```